mslib/msui/_tests/test_mscolab.py

```diff
@@ -521,6 +521,36 @@ def test_handle_leave_operation(self, mockmessage):
         assert self.window.listViews.count() == 0
         assert self.window.listOperationsMSC.model().rowCount() == 0
 
+    @mock.patch("PyQt5.QtWidgets.QMessageBox.information")
+    @mock.patch("PyQt5.QtWidgets.QInputDialog.getText", return_value=("new_name", True))
+    def test_handle_rename_operation(self, mockbox, mockpatch):
+        self._connect_to_mscolab()
```

Comment:
more tests ;) not related to the filter flight path. 
That would be better to seperate this to another PR next time ;)

Thx :)

Reply:
Okay, I'll take care of this next time.